### PR TITLE
changes in doc solveset --> substitution

### PR DESCRIPTION
**Pull request by MaqOwais**
Minor changes in doc's

#20613

* solveset --> substitution
  * improvement of  docs
  

NO ENTRY

### CODE REVIEW

File: sympy/solvers/solveset.py [hunk 1]
```diff
@@ -3213,9 +3213,9 @@ def _solve_using_known_values(result, solver):
     new_result_complex, solve_call2, cnd_call2 = _solve_using_known_values(
         old_result, solveset_complex)
 
-    # when `total_solveset_call` is equals to `total_conditionset`
-    # means solvest fails to solve all the eq.
-    # return conditionset in this case
+    # when `total_solveset_call` equals to `total_conditionset`
+    # implies `solveset` failed to solve all the equations
+    # returning conditionset in this case
```
Comment (oscarbenjamin) [inline sympy/solvers/solveset.py:3218]: Neither of these version is coherent English. I suggest:

If total_solveset_call is equal to total_conditionset then solveset failed to solve all of the equations. In this case we return a ConditionSet here.

The backticks are not needed in comments.

**Comment by oscarbenjamin** [comment]
A release note is not needed for a change to comments or for small change to docs.

**Comment by MaqOwais** [comment]
https://github.com/sympy/sympy/pull/20814#issuecomment-761928431

@oscarbenjamin please do reply.

**Comment by MaqOwais** [comment]
I think it should be merged ...

**Pull request merged**